src/client/sections/capability-section.tsx

```diff
@@ -43,7 +48,7 @@ export function CapabilitySection({ capability, route, additionalProps = {} }: S
           {deviceId && (
             <JWTCapability
               device={deviceId}
-              capability={`@transitive-robotics/${capability}`}
+              capability={`${_.find(fleet, { id: deviceId })?.capabilities[capability]?.scope}/${capability}`}
```

Comment:
Is this the right approach? Shouldn't the scope already be given as a prop to `CapabilitySection`? Presumably the calling context already knows which capabilities are running (including their scope), because otherwise that calling context shouldn't try to create it in first place.